pkg/job_controller/job.go

```diff
@@ -324,6 +333,103 @@ func (jc *JobController) ReconcileJobs(job interface{}, replicas map[apiv1.Repli
 	return result, nil
 }
 
+func (jc *JobController) ReconcileCron(meta metav1.Object, runtimeObj runtime.Object, policy *apiv1.RunPolicy) error {
+	cronJob := &appv1.Cron{}
+	// check
+	if policy == nil || policy.CronPolicy == nil{
```

Comment:
these two checks can be removed ? because it is already checked in the caller

Reply:
I double checked here specially.
if others call this funtion, I'm worried that othes forget to check